pkg/vm/vm_test.go

```diff
@@ -3331,6 +3331,132 @@ func TestBitAndNumericOpcodes(t *testing.T) {
 	}
 }
 
+var stackIsolationTestCases = map[opcode.Opcode][]struct {
+	name             string
+	hasDynamicInvoke bool
+	hasScriptOnStack bool
+	needSetRVCount   bool
+	offset           []byte
+	params           []interface{}
+}{
+	opcode.CALLE: {
+		{
+			name:             "no_params",
+			hasDynamicInvoke: false,
+			hasScriptOnStack: false,
+			needSetRVCount:   false,
```

Comment:
These bools are the same for every instruction, probably we can hardcode them into the test runner and not duplicate for each testcase here.